app/uk/gov/hmrc/apigatekeeperxmlservicesfrontend/views/organisation/OrganisationDetailsView.scala.html

```diff
@@ -36,10 +36,10 @@ <h1 class="govuk-heading-l">
       @organisation.name
     </dd>
     <dd class="govuk-summary-list__actions">
-      <!-- <a href="#" class="govuk-link--no-visited-state">
```

Comment:
Can this comment go?

Reply:
I thought the same, but realised that this change is to remove the comment!

Reply:
its a link that was hidden as we had';t implemented the manage functionality yet. The link is now active and visible.